app/models/concerns/as_localization.rb

```diff
@@ -63,9 +63,16 @@ def localize_in!(language)
 
   # standalone-category
   # parent-category/child-category
-  def slug_with_ancestors_slugs
+  def slug_with_ancestors_slugs(exclude_draft: true)
     slugs = about.ancestors_and_self.map do |ancestor|
-      ancestor.best_localization_for(language).slug
+      ancestor_l10n = ancestor.localization_for(language)
+      if ancestor_l10n.nil? || (exclude_draft && ancestor_l10n.try(:draft?))
+        # If l10n is nil or draft, no slug
+        nil
+      else
```

Comment:
@SebouChu si y a pas de l10n direct, on ne va plus chercher le best. Good or not?